en/docs/learn/api-gateway/passing-end-user-attributes-to-the-backend/passing-enduser-attributes-to-the-backend-using-jwt.md

```diff
@@ -125,7 +125,7 @@ if the <code>getClaims</code> method returns <code>{email:user1@wso2.com, gender
 <td><pre><code>apim.jwt.claim_dialect</code></pre></td>
 <td><div class="content-wrapper">
 <p>The dialect URI under which the user's claims are be looked for. Only works with the default value of the <code>apim.jwt.claims_extractor_impl</code> element defined above.</p>
-<p>The JWT token contains all claims define in the <code>apim.jwt.claim_dialect</code> element. The default value of this element is <code>http://wso2.org/claims</code>. To get a list of users to be included in the JWT, simply uncomment this element after enabling the JWT. It will include all claims in <code>http://wso2.org/claims</code> to the JWT token.</p>
+<p>The JWT token contains all claims that are defined in the <code>apim.jwt.claim_dialect</code> element. The default value of this element is <code>http://wso2.org/claims</code>. To get a list of user claims to be included in the JWT, simply uncomment this element after enabling the JWT. It will include all claims in <code>http://wso2.org/claims</code> to the JWT token.</p>
```

Comment:
is it still a list of user claims? or a list of a particular user's claims?

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/798/commits/b2c420afb90302c4d2f7cdfcba6653e538abcf0a